modules/apps/commerce/commerce-price-list-service/service.xml

```diff
@@ -104,7 +104,7 @@
 
 		<!-- Other fields -->
 
-		<column name="commerceCurrencyId" type="long" />
+		<column name="commerceCurrencyCode" type="String" />
 		<column name="parentCommercePriceListId" type="long" />
 		<column name="catalogBasePriceList" type="boolean" />
 		<column name="netPrice" type="boolean" />
```

Comment:
I fear BChan might have something to say about the lack of alphabetical order

Reply:
It is sorted. The difference is that the first 2 items are external references to other entries, all others are "normal" peoperties